### PR TITLE
Update rooms.json

### PR DESCRIPTION
**Pull request by kacycrider**
ready to go cap'n!

### CODE REVIEW

File: app/config/tstvhq/rooms.json [hunk 1]
```diff
@@ -249,12 +246,12 @@
...
-            "type": "CHATSTREAM",
+            "type": "JITSI",
             "capacity": 25,
             "muteRoom": false,
             "hideVideo": false,
             "description": "Meet your new best friend, a fruit bat or insect bat from BatWorld! Donate at batworld.org",
-            "backgroundImage": "https://tstvhq.s3.amazonaws.com/backgrounds/LUPERCALIA/LUPERCALIA-BG-belfry.png",
+            "backgroundImage": "https://tstvhq.s3.amazonaws.com/backgrounds/vhq/hexennacht/belfry.png",
             "iframeOptions": {
                 "src": "https://video.nest.com/embedded/live/UYYI6tFUy2?autoplay=1"
```
Comment (hndrewaall) [inline app/config/tstvhq/rooms.json:256]: if this is going to be a `JITSI` room we should get rid of the URL yes?

File: app/config/tstvhq/rooms.json [hunk 2]
```diff
@@ -279,8 +276,8 @@
             "muteRoom": false,
             "hideVideo": false,
             "description": "Give the goats from Woodstock Farm Sanctuary a snack at woodstocksanctuary.org/",
-            "type": "CHATSTREAM",
-            "backgroundImage": "https://tstvhq.s3.amazonaws.com/backgrounds/LUPERCALIA/LUPERCALIA-BG-sleeping-porch.png",
+            "type": "IFRAME",
```
Comment (hndrewaall) [inline app/config/tstvhq/rooms.json:279]: Is there a reason we want `IFRAME` and not `CHATSTREAM` here?

**Review by hndrewaall — CHANGES_REQUESTED** [review]
just a couple questions @kacycrider

File: app/config/tstvhq/rooms.json (hunk 2 above)
Comment (kacycrider) [reply app/config/tstvhq/rooms.json:279]: IFRAME because we want people to be able to talk in this room

File: app/config/tstvhq/rooms.json (hunk 1 above)
Comment (kacycrider) [reply app/config/tstvhq/rooms.json:256]: This should be IFRAME, i made mistake. revisin

Comment (hndrewaall) [reply app/config/tstvhq/rooms.json:256]: do you mean `CHATSTREAM`?

File: app/config/tstvhq/rooms.json (hunk 2 above)
Comment (hndrewaall) [reply app/config/tstvhq/rooms.json:279]: oic

File: app/config/tstvhq/rooms.json (hunk 1 above)
Comment (hndrewaall) [reply app/config/tstvhq/rooms.json:256]: i assume not per other comment

**Review by hndrewaall — APPROVED** [review]

**Pull request merged**